src/main/java/com/ericsson/ei/EndpointSecurity.java

```diff
@@ -85,8 +92,19 @@ private String decodeBase64(String password) {
     }
 
     private void addLDAPServersFromList(JSONArray serverList, AuthenticationManagerBuilder auth) throws Exception {
+        TextFormatter textFormatter = new TextFormatter();
+        StandardPBEStringEncryptor encryptor = new StandardPBEStringEncryptor();
+        encryptor.setPassword(jasyptEncryptorPassword);
         for (int i = 0; i < serverList.length(); i++) {
```

Comment:
Please add some linebreaks to logically seperate your code. Everything is bloated together and it becomes harder to read.

Reply:
Done